code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -77,6 +77,38 @@
 	new /obj/item/clothing/head/soft/sec/corp(src)
 	new /obj/item/clothing/under/rank/security/corp(src)
 
+/obj/structure/closet/secure_closet/soro/security
```

Comment:
don't make it a child of a non existent parent, should be like /soro_security or simply use inheritance properly and move /security into /security/standard to keep icon shit the same

Reply:
applied changes